web/src/components/ConfirmationModal/ConfirmationModal.tsx

```diff
@@ -0,0 +1,63 @@
+import { FC } from "react";
+
+import {
+  Dialog,
+  DialogTitle,
+  DialogContent,
+  DialogActions,
+  Theme,
+  Button,
+} from "@mui/material";
+import { useTheme } from "@mui/material/styles";
+
+interface Props {
+  showModal: boolean;
+  closeModal: () => void;
+  title: string;
+  onSubmit: () => void;
+}
+
+export const ConfirmationModal: FC<Props> = ({
+  children,
+  showModal,
+  closeModal,
+  title,
+  onSubmit,
+}) => {
+  const theme = useTheme<Theme>();
+
+  return (
+    <Dialog
+      onClose={closeModal}
+      open={showModal}
+      sx={{
+        "& .MuiPaper-root": {
+          width: "560px",
+        },
+      }}
+    >
+      <DialogTitle sx={{ fontSize: 24, lineHeight: "24px", pt: 2.5, pb: 1.75 }}>
+        {title}
+      </DialogTitle>
+      <DialogContent sx={{ pb: 7.5, color: theme.palette.secondary.dark }}>
+        {children}
```

Comment:
I have concerns here, as we're already setting `title` above, which could be children also.

I'd probably use `content` prop, to stay consistent with the `title`. Or would make "trick" like with the table, so force developer to use `Dialog.DialogTitle` & `Dialog.DialogContent` with some predefined props.